Xamarin.Forms.Core/NumericExtensions.cs

```diff
@@ -1,14 +1,18 @@
 using System;
+using System.ComponentModel;
 
-namespace Xamarin.Forms
+namespace Xamarin.Forms.Internals
 {
-	internal static class NumericExtensions
+	public static class NumericExtensions
 	{
+
+		[EditorBrowsable(EditorBrowsableState.Never)]
 		public static double Clamp(this double self, double min, double max)
 		{
 			return Math.Min(max, Math.Max(self, min));
 		}
 
+		[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Agreed. Could be. But let's save changing the public surface area for another PR. The goal of this PR is just to remove IVT and as we can see that's a pretty big change as is.